<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_paramconfig</class>
 <widget class="QDialog" name="Dialog_paramconfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>参数配置</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>287</width>
     <height>198</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>缆索总长</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_sensor1loc"/>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>传感器2位置</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_cablelength"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>传感器1位置</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_sensor2loc"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>声速补偿系数</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_speedcompensate"/>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="checkBox_sensor2enabled">
      <property name="text">
       <string>使能</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="checkBox_sensor1enabled">
      <property name="text">
       <string>使能</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_sensor3loc"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_sensor4loc"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>传感器3位置</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>传感器4位置</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>cm</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QCheckBox" name="checkBox_sensor3enabled">
      <property name="text">
       <string>使能</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QCheckBox" name="checkBox_sensor4enabled">
      <property name="text">
       <string>使能</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_paramconfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_paramconfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
